src/main/resources/flyway/conf/flyway.conf

```diff
@@ -134,6 +134,7 @@ flyway.locations = filesystem:src/main/resources/flyway/sql,classpath:db.migrati
 # Placeholders to replace in Sql migrations
 # flyway.placeholders.user=
 # flyway.placeholders.my_other_placeholder=
+flyway.placeholders.default_provider = GOOGLE
```

Comment:
i'm not sure if this is because our flyway implementation or just Flyway itself, but not having the value here causes an error when running  the `.fly` script: 
`No value provided for placeholder expressions: ${default_provider}.  Check your configuration!`